### PR TITLE
Add libxmlb 0.3.11

### PR DESCRIPTION
**Pull request by taozuhong**

### CODE REVIEW

File: mingw-w64-libxmlb/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+_realname=libxmlb
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.3.11
+pkgrel=1
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
+pkgdesc="A library to help create and query binary XML blobs (mingw-w64)"
+depends=("${MINGW_PACKAGE_PREFIX}-glib2"
+         "liblzma"
```
Comment (Biswa96) [inline mingw-w64-libxmlb/PKGBUILD:12]: Shouldn't these also have the MINGW_PACKAGE_PREFIX ?

File: mingw-w64-libxmlb/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+         "libzstd")
+makedepends=("${MINGW_PACKAGE_PREFIX}-gobject-introspection"
+             "${MINGW_PACKAGE_PREFIX}-meson"
+             "${MINGW_PACKAGE_PREFIX}-pkgconf"
+             "${MINGW_PACKAGE_PREFIX}-vala"
+             "${MINGW_PACKAGE_PREFIX}-cc")
+options=('strip' 'staticlibs')
```
Comment (Biswa96) [inline mingw-w64-libxmlb/PKGBUILD:19]: These options are default and not required here. Default options can be found in /etc/makepkg_mingw.conf file for mingw packages.

File: mingw-w64-libxmlb/PKGBUILD (hunk 1 above)
Comment (taozuhong) [reply mingw-w64-libxmlb/PKGBUILD:12]: they are MSYS package, not the MINGW package, so they don't have prefix.

Comment (taozuhong) [reply mingw-w64-libxmlb/PKGBUILD:12]: But it depends on them, how should I do?

**Comment by lazka** [comment]
Closing in favor of #17956

**Pull request closed without merging**